### DISCUSSION TITLE
When a text is pasted, editor sometimes dont identify the font family and size and related attributes

### DISCUSSION THREAD

**Discussion by SarathSantoshDamaraju** [ueberdosis/tiptap#3066] [General Questions]
The need is to display the fontfamily and the fontsize of the pasted text. 

I use Editor [events](https://tiptap.dev/api/events#list-of-available-events) to determine the attributes of the pasted text. Like `font-family` `font-size` to be a few essentials to start with.

But i observed that, most of the times the values are not being determined correctly.  

Is there any correct way to do this ?

Ref: https://codesandbox.io/s/youthful-grothendieck-t166cd?file=/src/App.vue

**Comment by sonsu-lee** [comment]
I have a similar problem....
Has this issue been resolved?

**Comment by nalaso** [comment]
You also need to use Color and Font-Family extension to achieve this 

```javascript
const editor =new Editor({
      extensions: [
        StarterKit,
        TextStyle,
        Color.configure({
          types: ["textStyle"],
        }),
        FontFamily.configure({
          types: ["textStyle"],
        }),
      ]
})
``` 

Also I am not sure whether tiptap support all font family

here is an example 
[https://codesandbox.io/s/infallible-hopper-7z2559?file=/src/App.vue](https://codesandbox.io/s/infallible-hopper-7z2559?file=/src/App.vue)
